core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -9,7 +9,12 @@ import com.unciv.models.translations.getPlaceholderText
 // I didn't put this in a companion object because APPARENTLY doing that means you can't use it in the init function.
 private val numberRegex = Regex("\\d+$") // Any number of trailing digits
 
-enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags: List<UniqueFlag> = emptyList()) {
+enum class UniqueType(
+    val text: String,
+    vararg targets: UniqueTarget,
+    val flags: List<UniqueFlag> = emptyList(),
+    val docDescription: String? = null
```

Comment:
Hmm, I see myself adding lots of these, every time there's a modder question about a speciific unique that's unanswered in current docs, good work!